operation-common/src/main/java/org/sopt/makers/operation/code/failure/auth/AuthFailureCode.java

```diff
@@ -0,0 +1,27 @@
+package org.sopt.makers.operation.code.failure.auth;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import org.sopt.makers.operation.code.failure.FailureCode;
+import org.springframework.http.HttpStatus;
+
+import static org.springframework.http.HttpStatus.BAD_REQUEST;
+import static org.springframework.http.HttpStatus.NOT_FOUND;
+
+@Getter
+@RequiredArgsConstructor
+public enum AuthFailureCode implements FailureCode {
+    // 400
+    NOT_NULL_PARAMS(BAD_REQUEST, "쿼리 파라미터 중 데이터가 들어오지 않았습니다."),
+    INVALID_SOCIAL_TYPE(BAD_REQUEST, "유효하지 않은 social type 입니다."),
+    INVALID_ID_TOKEN(BAD_REQUEST, "유효하지 않은 id token 입니다."),
+    INVALID_SOCIAL_CODE(BAD_REQUEST, "유효하지 않은 social code 입니다."),
+    FAILURE_READ_PRIVATE_KEY(BAD_REQUEST, "Private key 읽기 실패"),
+    // 404
```

Comment:
`P4`

개행하면 조금 더 가독성 좋을 것 같습니다!!
```suggestion
    FAILURE_READ_PRIVATE_KEY(BAD_REQUEST, "Private key 읽기 실패"),
    
    // 404
```